packages/@react-types/radio/src/index.d.ts

```diff
@@ -26,17 +26,27 @@ import {
 import {ReactElement, ReactNode} from 'react';
 
 export interface RadioGroupProps extends ValueBase<string>, InputBase, Validation, LabelableProps {
+  /**
+   * The Radio(s) contained within the RadioGroup.
+   */
   children: ReactElement<RadioProps> | ReactElement<RadioProps>[],
   /**
    * The axis the Radio Button(s) should align with.
    * @default 'vertical'
    */
   orientation?: Orientation,
-  name?: string // HTML form name. Not displayed.
+  /**
+   * The unique identifying name of the RadioGroup that is applied to the Radios within the group. It is not a displayed value.
+   * See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name_and_radio_buttons).
+   */
+  name?: string
 }
 
 export interface RadioProps extends FocusableProps {
-  value: string, // HTML form value. Not displayed.
+  /**
+   * The value of the radio button, used to identify it in a RadioGroup. It is not a displayed value.
+   */
+  value: string,
```

Comment:
probably match more closely to the one above for name and value?
https://github.com/adobe-private/react-spectrum-v3/pull/535/files#diff-ffcbdbe81b263a37158c30d2ef607220R42